packages/client/src/StreamrClient.ts

```diff
@@ -659,6 +659,13 @@ export class StreamrClient {
         return (await this.node.getNode()).getDiagnosticInfo()
     }
 
+    /**
+     * @deprecated This in an internal method
```

Comment:
It can be public if we want. 

Earlier we've annotated `StreamrClient#getNode` as internal because we don't want end-users to access the network via the low level methods of `NetworkNodeStub`. Similarly the config defaults (e.g. the url the of The Graph) are very low level details, which end-users should never need.
https://github.com/streamr-dev/network/blob/6fde5ebde7af6c382a6d343a307da09f2d4bd28a/packages/client/src/StreamrClient.ts#L571